gapis/api/gles/draw_call_mesh.go

```diff
@@ -46,9 +46,16 @@ func drawCallMesh(ctx context.Context, dc drawCall, p *path.Mesh) (*api.Mesh, er
 
 	c := GetContext(s, dc.Thread())
 
-	dci, err := dc.getIndices(ctx, c, s)
-	if err != nil {
-		return nil, err
+	noData := p.Options != nil && p.Options.ExcludeData
+
+	var dci drawCallIndices
+	if noData {
+		dci = drawCallIndices{nil, dc.getDrawMode(), false}
```

Comment:
If we're not getting data, is the draw mode important? If not, can we drop the new interface method?